### PR TITLE
miami_header

### PR DESCRIPTION
**Pull request by postrygan**
https://postrygan.github.io/layout_miami/

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  transition: transform 0.3s;
+}
+
+.page__menu:target {
+  transform: translate(0);
+}
+
+body {
+  background: #eee;
+  margin: 0;
```
Comment (SerhiiKirik) [inline src/styles/main.css:268]: move to normalize file

```suggestion
body {
  background: #eee;
```

File: src/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+  text-align: left;
+}
+
+.page__menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  transform: translateX(-100%);
+  transition: transform 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/main.css:259]: create variable for this
```suggestion
  transition: transform $transitionDuration;
```

File: src/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+  font-weight: 700;
+  line-height: 16px;
+  letter-spacing: 1.5px;
+  text-transform: uppercase;
+}
+
+.locations-phone {
+  width: 201px;
+  height: 108px;
+  color: #fff;
```
Comment (SerhiiKirik) [inline src/styles/main.css:229]: create variables for all colors
```suggestion
  color: $c-white;
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done 🚀 
But, please, deploy you changes with npm run deploy
Also, fix comments

**Review by anstsot — CHANGES_REQUESTED** [review]
The demo link isn't working. please deploy your changes
image

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Good start :+1: 
Please re-request review when entire task will be done
If you have any questions feel free to ask in chat

**Pull request closed without merging**